api/src/main/resources/workbench-api.yaml

```diff
@@ -2432,6 +2432,41 @@ paths:
           description: The Terra Job for the extraction workflow
           schema:
             "$ref": "#/definitions/GenomicExtractionJob"
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/cohorts/abortExtract/{wgsCohortExtractionJobId}":
+    post:
+      parameters:
+        - in: path
+          name: workspaceNamespace
+          description: workspaceNamespace
+          required: true
+          type: string
+        - in: path
+          name: workspaceId
+          description: workspaceId
+          required: true
+          type: string
+        - in: path
+          name: wgsCohortExtractionJobId
+          type: string
+          required: true
+      tags:
+        - dataSet
+      description: 'Aborts a running genome extract job. Requires WRITER access on the workspace.
+        Workspace CDR must also have access to a Wgs dataset'
+      operationId: abortExtract
```

Comment:
I would probably name this: `abortGenomicExtractionJob`

for consistency with the "list jobs" endpoint above